pkg/virt-launcher/virtwrap/network/common.go

```diff
@@ -346,17 +348,32 @@ func (h *NetworkUtilsHandler) GenerateRandomMac() (net.HardwareAddr, error) {
 }
 
 func (h *NetworkUtilsHandler) CreateTapDevice(tapName string, isMultiqueue bool, launcherPID string) error {
-	tapDeviceArgs := []string{"tuntap", "add", "mode", "tap", "user", "qemu", "group", "qemu", "name", tapName}
+	launcherSELinuxLabel, err := selinux.FileLabel(fmt.Sprintf("/proc/%s/attr/current", launcherPID))
```

Comment:
Can you split the selinux stuff from the creation of the tap? (e.g. extract labeling and friends to a helper)

Reply:
I can.